new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElkSample.java

```diff
@@ -0,0 +1,122 @@
+
+  // constants for ELK Stack
+  private static LoggingExporterParams elasticsearchParams = null;
+  private static LoggingExporterParams kibanaParams = null;
+  private static LoggingFacade logger = null;
+
+  /**
+   * Install Elasticsearch, Kibana and Operator, create a one cluster domain.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void init(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+    // create standard, reusable retry/backoff policy
+    withStandardRetryPolicy = with().pollDelay(2, SECONDS)
+      .and().with().pollInterval(10, SECONDS)
+      .atMost(5, MINUTES).await();
+
+    // get a new unique opNamespace
+    logger.info("Assigning a unique namespace for Operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    // get a new unique domainNamespace
+    logger.info("Assigning a unique namespace for Domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    // install and verify Elasticsearch
+    logger.info("install and verify Elasticsearch");
+    elasticsearchParams = assertDoesNotThrow(() -> installAndVerifyElasticsearch(),
+            String.format("Failed to install Elasticsearch"));
+    assertTrue(elasticsearchParams != null, "Failed to install Elasticsearch");
+
+    // install and verify Kibana
+    logger.info("install and verify Kibana");
+    kibanaParams = assertDoesNotThrow(() -> installAndVerifyKibana(),
+        String.format("Failed to install Kibana"));
+    assertTrue(kibanaParams != null, "Failed to install Kibana");
+
+    // install and verify Operator
+    installAndVerifyOperator(opNamespace, opNamespace + "-sa", false, 0, true, domainNamespace);
+  }
+
+  /**
+   * Uninstall ELK Stack and delete domain custom resource.
+   */
+  @AfterAll
+  void tearDown() {
+    // uninstall ELK Stack
+    if (elasticsearchParams != null) {
+      logger.info("Uninstall Elasticsearch pod");
+      assertDoesNotThrow(() -> uninstallAndVerifyElasticsearch(elasticsearchParams),
+          "uninstallAndVerifyElasticsearch failed with ApiException");
+    }
+
+    if (kibanaParams != null) {
+      logger.info("Uninstall Elasticsearch pod");
+      assertDoesNotThrow(() -> uninstallAndVerifyKibana(kibanaParams),
+          "uninstallAndVerifyKibana failed with ApiException");
+    }
+  }
+
+  /**
+   * Verify that ELK Stack is ready to use by checking the index status of
+   * Kibana and Logstash creaed in the Operator pod successfully.
+   */
+  @Test
+  @DisplayName("Verify that ELK Stack is ready to use")
+  public void testLogLevelSearch() throws Exception {
+    // Verify that Elastic Stack is ready to use
+    verifyLoggingExporterReady(opNamespace, null, LOGSTASH_INDEX_KEY);
```

Comment:
Instead of adding single opaque Method (verifyLoggingExporterReady) need to provide small methods for each steps in the the usescase we are trying to verify.  This will be easy to triage by going step-by-step when the test fails

Reply:
This java class is not really test anything. The purpose of the class is to make sure the ELK Stack is installed successfully. I may remove this class after the PR is merged. For real test cases, we have junit5-elk-tests, PR https://github.com/oracle/weblogic-kubernetes-operator/pull/1848